src/operator/numpy/np_elemwise_unary_op_basic.cc

```diff
@@ -133,6 +133,7 @@ Example::
 // abs
 MXNET_OPERATOR_REGISTER_NUMPY_UNARY(_npi_absolute, "x", mshadow_op::abs)
 .add_alias("_npi_abs")
+.add_alias("_npi_fabs")
```

Comment:
seems like you're only using `npi.abs` in your Python wrappers, then no need for this alias.